src/ddmd/expression.d

```diff
@@ -8413,6 +8413,22 @@ extern (C++) final class ImportExp : UnaExp
             ob.writestring(")");
             ob.writenl();
         }
+        if (global.params.makeDeps !is null)
```

Comment:
Hmm CodeCov says this is tested:

![image](https://user-images.githubusercontent.com/4370550/27844951-632f38d0-6129-11e7-9f00-9bc086628540.png)